lighthouse-plugin-publisher-ads/audits/ad-blocking-tasks.js

```diff
@@ -82,18 +106,21 @@ class AdBlockingTasks extends Audit {
     try {
       longTasks = await LongTasks.request(metricData, context);
     } catch (e) {
-      return auditNotApplicable(NOT_APPLICABLE.INVALID_TIMING);
+      return auditNotApplicable(
+        str_(common.UIStrings.NOT_APPLICABLE__INVALID_TIMING));
     }
 
     if (!longTasks.length) {
-      return auditNotApplicable(NOT_APPLICABLE.NO_TASKS);
+      return auditNotApplicable(
+        str_(common.UIStrings.NOT_APPLICABLE__NO_TASKS));
     }
 
     const {timing: endTime} =
         await AdRequestTime.request(metricData, context);
 
     if (!(endTime > 0)) { // Handle NaN, etc.
-      return auditNotApplicable(NOT_APPLICABLE.NO_AD_RELATED_REQ);
+      return auditNotApplicable(
```

Comment:
Instead of passing in arguments to `auditNotApplicable`, it could be cleaner to define methods for each error type. Like:

```
  return auditNotApplicable.NoAdRelatedRequest();
```

Reply:
Great idea, done.